pkg/zonemgr/zone_manager_test.go

```diff
@@ -1,6 +1,7 @@
 package zonemgr
```

Comment:
If possible please move the test to `zonemgr_test` package.

Reply:
fixed